README.md

```diff
@@ -42,10 +42,10 @@ Promote awareness about your CFP in the community:
 
 # Getting a design approved
 
-For a CFP to be considered viable, a Cilium committer needs to aprove it.
-After the approval, the design can be merged. A merged design proposal 
-means the proposal is viable to be executed on, but not that there is a
-100% chance it will be accepted.
+For a CFP to be considered implementable, a Cilium committer needs to aprove it.
+After their approval, the design can be merged. A merged design proposal 
```

Comment:
One of the reasons in other projects we merge proposals in progress and proposals that never make it is as a form of documentation on conversations on why/why the project did something.

I suggest we allow anything that has substantial conversations to be documented in a CFP (i.e. merged) -- even if in the end we decide not to go ahead.

Reply:
(see my comment below with respect to always filing a feature).

Reply:
I'd like us to encourage this. :+1:

Reply:
I agree with that too, is there anything else you think we should add to the wording here?

Reply:
added some suggested wording about dormant and declined also being important and how they can be merged, let me know what you think